tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/Checks/ProcessBasicChecksTests.cs

```diff
@@ -143,6 +141,10 @@ public async Task WrongEnvironmentVariables()
                 MissingProfilerEnvironment(CorProfilerPath64Key, "dummyPath"),
                 WrongProfilerEnvironment(CorProfilerPath64Key, "dummyPath"));
 
+            // Because TracingWithInstaller has a long URL, it gets split in the Spectre.Console output
```

Comment:
The test passed on my PR without removing spaces, should I backport the change?

Reply:
Yeah that's because you had an integration test (which just invokes the code) and I transformed it to an artifact test (which actually call the executable). When doing that, Spectre.Console adds line breaks to the output which mess up with the assertions.